src/app/components/film-dialog/dialog/delete-dialog.js

```diff
@@ -0,0 +1,41 @@
+import {deleteFilm, loadFilms, setConfirmationDialog, setResultPopup, setSelectedFilm} from "../../../store/slices";
+import ConfirmationDialog from "../../../components/general/confimation-dialog/confirmation-dialog";
+import {selectDialogs, selectSelectedFilm} from "../../../store/selectors";
+import {useDispatch, useSelector} from "react-redux";
+import React from "react";
+
+export default function DeleteDialog() {
+    const selectedFilm = useSelector(selectSelectedFilm);
+    const dialogs = useSelector(selectDialogs);
+    const dispatch = useDispatch();
+
+    const onDelete = () => {
+        dispatch(deleteFilm(selectedFilm,
+            (dispatch) => {
+                dispatch(setResultPopup({
+                    title: 'Congratulations!',
+                    description: `The movie has been deleted successfully`,
+                    type: 'Success'
+                }));
+                dispatch(loadFilms());
+                dispatch(setSelectedFilm(null));
```

Comment:
Not sure, but it is possible to use fewer dispatchers to handle this in a single action?